app/models/picture.rb

```diff
@@ -0,0 +1,10 @@
+class Picture < ActiveRecord::Base
+
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  has_attached_file :image, :styles => { :medium => "500x500>", :thumb => "100x100>" }, :default_url => "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [132/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)